### ISSUE TITLE
Is 1.0.2 going to be officially released any time soon?

### ISSUE THREAD

**Issue by GeorgeJahad** [brutasse/graphite-api#101]
We, (the Blueflood team at Rackspace,) have been happy using release 1.0.1 and have added a finder for it.

Now we would like to add fetch_multi support, (which is not in 1.0.1.)  Is 1.0.2 going to be officially released any time soon? If not, is there a particular commit you recommend I use?  or should I just take the HEAD of the master branch?

Thanks much for the great work!

**Comment by brutasse** [comment]
HEAD is considered stable so you can definitely go ahead and deploy it. There's an FPM script if you want to build a deb package.

That said you're right, I really need to make a new release… Keeping this open in the meantime :)

**Comment by inthecloud247** [comment]
+1 cool.

**Comment by cbeer** [comment]
:+1: `master` fixes some annoying rendering bugs for me.

**Comment by hggh** [comment]
:+1:

**Comment by schaubl** [comment]
:+1:

**Comment by jamtur01** [comment]
This would be great - new DEB package would be awesome as #42 has broken the existing package.

**Comment by hggh** [comment]
@jamtur01 the Graphite Team inside Debian is planing to upload Graphite-API into Debian, after a new release of graphite-api is released.

**Comment by jamtur01** [comment]
@hggh - any idea on an ETA on either? Thanks!

**Comment by filippog** [comment]
@hggh :+1: I'd be interested too to see graphite-api in Debian!

**Comment by daks** [comment]
+1 for a new release :)

**Comment by hggh** [comment]
@brutasse are you going to release a new version of graphite-api soon?

**Comment by nikut** [comment]
@brutasse Is current HEAD actually supposed to be stable like you commented in March? 

I tried upgrading to it from 1.0.1 mainly to get Carbonlink cache query support and I cannot get it working. The only queries that (mostly) work are ones getting the Carbon internal metrics, everyhing else fails with a 500 error, even when testing locally with curl. Is there something obvious I'm missing or is current HEAD just broken?

**Comment by petarmaric** [comment]
Do you guys need any help with testing or releasing the new version? I'm a bit busy this semester, but I could probably find some time...

**Comment by brutasse** [comment]
@nikut HEAD is supposed to be stable. Do you have more information about these 500 errors? Which version of Carbon do you run?

@petarmaric testing is more than welcome if you're up for it :)

**Comment by nikut** [comment]
@brutasse I would love to provide more information on the errors but I cannot figure out how to get graphite-api to log stack traces or anything like that. Any tips? I'm not seeing any errors in graphite-api.log with the following logging configuration (adapted from some example somewhere without really understanding how the logging configuration works):

```
logging:
  version: 1
  disable_existing_loggers: false
  handlers:
    file:
      class: logging.FileHandler
      filename: /logs/graphite/logs/graphite-api.log
      level: DEBUG
  loggers:
    root:
      handlers:
        - file
      propagate: true
      level: DEBUG
    graphite_api:
      handlers:
        - file
      propagate: true
      level: DEBUG
```

I'm running Carbon 0.9.13. 

Should I open a separate issue for these errors? Thanks!

**Comment by runningman84** [comment]
I would really appricate a release.

**Comment by petarmaric** [comment]
@brutasse Sure, I'll try to help as much as I can. Any ideas on where I could to start?

**Comment by brutasse** [comment]
I just released graphite-api 1.1: https://github.com/brutasse/graphite-api/releases

**Comment by nikut** [comment]
@brutasse Awesome, thank you! Based on a quick test it seems that my issue with the HEAD version failing on queries has been fixed in 1.1, so thanks for that, too.

**Comment by daks** [comment]
thanks @brutasse
